sklearn/feature_extraction/text.py

```diff
@@ -796,7 +796,7 @@ def _document_frequency(X):
 
 
 class CountVectorizer(_VectorizerMixin, BaseEstimator):
-    """Convert a collection of text documents to a matrix of token counts
+    r"""Convert a collection of text documents to a matrix of token counts
```

Comment:
Is it possible to fix the `token_pattern` without making the docstring raw?

Reply:
I have tried a number of combinations of `\` and `\\` but the `\b` just disappear if the docstring is not raw. I can't claim this is the only solution, maybe there is something I couldn't figure out...